### PR TITLE
fix #86

### LINKED ISSUE #86: Scipion is not awere of the status of submitted jobs
**Migrated from**: https://github.com/I2PC/scipion/issues/2080
**Posted by**: @JuhaHuiskonen 

Sometimes submission of the job to a scheduling system fails (for example when the user asks for more memory than allowed in a particular queue). Scipion shows this job with status "Launched". It would be nice if Scipion kept checking if the job is actually in the queue or not (if it got a job ID or not). If not, it should be marked as "Failed"

### PR DESCRIPTION
**Pull request by azazellochg**

### CODE REVIEW

File: pyworkflow/project/project.py [hunk 1]
```diff
@@ -634,7 +634,9 @@ def launchProtocol(self, protocol, wait=False, scheduled=False,
             pwutils.path.copyFile(self.dbPath, protocol.getDbPath())
 
         # Launch the protocol, the jobId should be set after this call
-        pwprot.launch(protocol, wait)
+        jobId = pwprot.launch(protocol, wait)
+        if jobId is None or jobId == -1:
```
Comment (pconesa) [inline pyworkflow/project/project.py:638]: Should we use UNKNOWN_JOBID instead of -1 ?

**Comment by azazellochg** [comment]
Sure!

File: pyworkflow/project/project.py (hunk 1 above)
Comment (azazellochg) [reply pyworkflow/project/project.py:638]: it's not defined in constants, just in one file. Do you want to move it to constants?

Comment (pconesa) [reply pyworkflow/project/project.py:638]: I do not see it necessary if used within the same file, for me is fine to be at the top. But I'm ok if you want to move it there.

**Comment by azazellochg** [comment]
@pconesa ready to merge!

**Comment by pconesa** [comment]
I'm just realizing that the constant is at launch.py. I'll reuse the same.

**Comment by pconesa** [comment]
@azazellochg, do you agree with this?

**Comment by azazellochg** [comment]
yep! all good!

**Pull request merged**